integrations/deploy.yaml

```diff
@@ -188,7 +188,7 @@
     link: ''
     categories:
       - deploy.operating-systems
-    icon_filename: 'apple.png'
+    icon_filename: 'apple.svg'
```

Comment:
the current is `macos.svg`

Reply:
I think the MacOS one doesn't seem right
![image](https://github.com/netdata/netdata/assets/82235632/54bc44b4-48f7-44cf-b074-e1d58f3bc69c)

We have 3 icons for Mac/apple, the othe two the black one is a png and the third is the svg (not added by me was already there)
![image](https://github.com/netdata/netdata/assets/82235632/514538b0-340f-4bc2-a224-b6cbc8ebaa11)

![image](https://github.com/netdata/netdata/assets/82235632/615a301d-4e2e-475b-847b-b828948d7fe5)

so I would go with the grey one and we will need to fix the `macos.svg` to `apple.svg`

cc/ @sashwathn @shyamvalsan

Reply:
> This PR is [28 commits behind](https://github.com/netdata/netdata/compare/hugo-fix-data-collection-icons-1...master) master.

@ilyam8 I think this is fixed, right?

Reply:
looks like it, now it includes only the yamls, not past commits

Reply:
(removing the guys that were dragged in here due to no rebase)